src/main/java/org/openpnp/vision/pipeline/CvPipeline.java

```diff
@@ -197,6 +198,14 @@ public void setNozzle(Nozzle nozzle) {
     public Nozzle getNozzle() {
         return nozzle;
     }
+	
+    public void setValue(double value) {
```

Comment:
@cri-s I didn't ask you to remove the "closed loop and other functionality". I'm asking you to remove setValue() which does not appear to be used. If it's not used, then what is the value of adding it? And if it *is* used, then commit the code that uses it. Right now none of the code you submitted uses it.